libethcore/Common.h

```diff
@@ -90,6 +96,13 @@ enum class RelativeBlock: BlockNumber
 	Pending = PendingBlock
 };
 
+enum class BlockPolarity
```

Comment:
This is used only in `LocalisedLogEntry`, maybe move it to `LogEntry.h`?

Reply:
Unfortunately, this is also used in Client.cpp and ClientBase.cpp.